parse_trending_models.py

```diff
@@ -1,65 +1,350 @@
-from huggingface_hub import HfApi
-import os
-from datetime import datetime
+import argparse
+from huggingface_hub import HfApi, file_exists, upload_file
 import json
+from datetime import datetime
+import requests
+import os
+from dotenv import load_dotenv
+import logging
+from typing import Dict, List, Any, Optional
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+)
+logger = logging.getLogger(__name__)
+
+load_dotenv()
+
+AVOCADO_AUTHORS = [
+    "ariG23498",
+    "reach-vb",
+    "pcuenq",
+    "burtenshaw",
+    "dylanebert",
+    "davanstrien",
+    "merve",
+    "sergiopaniego",
+    "Steveeeeeeen",
+    "ThomasSimonini",
+]
+
+
+class ModelChecker:
+    def __init__(self, debug: bool = False, limit: int = 100):
+        self.debug = debug
+        self.limit = limit
+
+        self.hf_api = HfApi()
+        self.data_folder = "trending_data"
+        self.repo_id = "ariG23498/trending_models"
```

Comment:
let's move this to an org, where we can add people and even make it private if we wanted to

Reply:
Do you have any suggestions?

Reply:
yes, as I said, move it to an org so that other people can interact with it. we can just make a new org.